src/frontend/running.rs

```diff
@@ -28,7 +34,7 @@ pub enum RunningInput {
         best_block_number: BlockNumber,
         reward_address_balance: Balance,
         initial_farm_states: Vec<InitialFarmState>,
-        raw_config: RawConfig,
+        raw_config: Box<RawConfig>,
```

Comment:
Why moving onto the heap?

Reply:
Otherwise, there is a clippy warning:
```
warning: large size difference between variants
  --> src/frontend/running.rs:36:1
   |
36 | /  pub enum RunningInput {
37 | |/     Initialize {
38 | ||         best_block_number: BlockNumber,
39 | ||         reward_address_balance: Balance,
40 | ||         initial_farm_states: Vec<InitialFarmState>,
...  ||
43 | ||         chain_constants: ChainConstants,
44 | ||     },
   | ||_____- the largest variant contains at least 268 bytes
45 | |      NodeNotification(NodeNotification),
   | |      ---------------------------------- the second-largest variant contains at least 48 bytes
...  |
48 | |      TogglePausePlotting,
49 | |  }
   | |__^ the entire enum is at least 0 bytes
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant
   = note: `#[warn(clippy::large_enum_variant)]` on by default
help: consider boxing the large fields to reduce the total size of the enum
   |
41 |         raw_config: Box<RawConfig>,
   |                     ~~~~~~~~~~~~~~
   ```

Reply:
Please suppress this one specifically